### PR TITLE
Fixes #1141: Add option to change user roles from admin

### LINKED ISSUE #1141: Add option to change User Roles in the Admin section
**Actual Behaviour**
* Not present

**Expected Behaviour**
* Add option to change User Roles in the Admin section via a Edit button.

Yes

### PR DESCRIPTION
**Pull request by akshatnitd**
Fixes #1141 

Changes: [Add here what changes were made in this issue and if possible provide links.]
*Add option to change user roles from admin
Surge Deployment Link: https://pr-1177-fossasia-susi-skill-cms.surge.sh

### CODE REVIEW

File: src/components/Admin/ListUser/ListUser.js [hunk 1]
```diff
@@ -184,10 +283,121 @@ export default class ListUser extends Component {
...
+                  marginLeft: '-20px',
+                }}
+                autoWidth={false}
+              >
+                <MenuItem
+                  primaryText="BOT"
+                  value="bot"
+                  className="setting-item"
+                />
+                <MenuItem
```
Comment (Akshat-Jain) [inline src/components/Admin/ListUser/ListUser.js]: Please remove the option to change user role to `ANONYMOUS`.

File: src/components/Admin/ListUser/ListUser.js [hunk 2]
```diff
@@ -184,10 +283,121 @@ export default class ListUser extends Component {
...
+                  value="accountcreator"
+                  className="setting-item"
+                />
+                <MenuItem
+                  primaryText="ADMIN"
+                  value="admin"
+                  className="setting-item"
+                />
+                <MenuItem
+                  primaryText="BUREAUCRAT"
```
Comment (Akshat-Jain) [inline src/components/Admin/ListUser/ListUser.js]: `BUREAUCRAT` has been changed to `SUPERADMIN`. Please make the change.

File: src/components/Admin/ListUser/ListUser.js [hunk 3]
```diff
@@ -184,10 +283,121 @@ export default class ListUser extends Component {
...
+                  value="user"
+                  className="setting-item"
+                />
+                <MenuItem
+                  primaryText="REVIEWER"
+                  value="reviewer"
+                  className="setting-item"
+                />
+                <MenuItem
+                  primaryText="ACCOUNTCREATOR"
```
Comment (Akshat-Jain) [inline src/components/Admin/ListUser/ListUser.js]: `ACCOUNTCREATOR` has been changed to `OPERATOR`. Please make the change.

**Comment by Akshat-Jain** [comment]
@akshatnitd Looks good, but please check https://github.com/fossasia/susi_skill_cms/issues/1184. Thanks :-)

**Comment by DNS-404** [comment]
We need approval of mentors before proceeding with #1184.

**Pull request merged**